automation_testing/pages/FilterComponent.ts

```diff
@@ -0,0 +1,109 @@
+import { Pages } from "../pages/Pages";
```

Comment:
Please pull from develop and push again. This file has already been reviewed in another PR.